packages/itwin/tree-widget/src/components/trees/models-tree/ModelsTree.tsx

```diff
@@ -38,10 +39,16 @@ export interface ModelsTreeHierarchyConfiguration {
 }
 
 /**
- * Props for [[ModelsTree]] component
+ * Base props for [[ModelsTree]] component.
  * @public
  */
-export interface ModelsTreeProps extends BaseFilterableTreeProps {
+export type ModelsTreeBaseProps = BaseFilterableTreeProps & TreeContextMenuProps;
```

Comment:
do we need  / use this type anywhere besides using it below?

Reply:
No it used as base type for `ModelsTreeProps`.

Reply:
Applied suggestion